Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorNodes.h

```diff
@@ -116,7 +116,6 @@ namespace AZ::DocumentPropertyEditor::Nodes
         static constexpr auto RemoveNotify = CallbackAttributeDefinition<void(size_t index)>("RemoveNotify");
         static constexpr auto ClearNotify = CallbackAttributeDefinition<void()>("ClearNotify");
         static constexpr auto ContainerCanBeModified = AttributeDefinition<bool>("ContainerCanBeModified");
-        static constexpr auto PromptOnContainerClear = AttributeDefinition<bool>("PromptOnContainerClear");
```

Comment:
Is there a reason we didn't keep this functionality? Correct there currently isn't a consumer of this attribute, so would be easy to add back in.

Reply:
yeah, it's just because it's an unused attribute that we now have to forward through multiple spots. I think it's generally bad design to implement a feature before it's actually used.